hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/log/LogLevel.java

```diff
@@ -340,22 +337,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response
         out.println(MARKER
             + "Submitted Class Name: <b>" + logName + "</b><br />");
 
-        Log log = LogFactory.getLog(logName);
+        Logger log = Logger.getLogger(logName);
         out.println(MARKER
             + "Log Class: <b>" + log.getClass().getName() +"</b><br />");
         if (level != null) {
           out.println(MARKER + "Submitted Level: <b>" + level + "</b><br />");
         }
 
-        if (log instanceof Log4JLogger) {
-          process(((Log4JLogger)log).getLogger(), level, out);
-        }
-        else if (log instanceof Jdk14Logger) {
-          process(((Jdk14Logger)log).getLogger(), level, out);
-        }
-        else {
-          out.println("Sorry, " + log.getClass() + " not supported.<br />");
-        }
+        process(log, level, out);
```

Comment:
still need to handle situations (logback..) where the logger is unknown/unsupported, right?

Reply:
Since we are directly instantiating `org.apache.log4j Logger` instance, we don't need to check for unknown logger I believe. 

If we can't set appropriate log type to logger, we would anyways this in the `process()`:
```
out.println(MARKER + "Bad Level : <b>" + level + "</b><br />")
```

Hence, i think we should be good here.

Reply:
The plan is to use SLF4J here, but we don't have hadoop-logging module yet. When I work on the sub-task to create hadoop-logging module, with sufficient reflections, we should be able to do `log.setLevel()` using the module. Hence, replacing this with slf4j would be taken care of with it.

Reply:
I will work on using SLF4J here with HADOOP-18207 sub-task.